runtime/manifest-parser.peg

```diff
@@ -27,6 +27,9 @@
     if (['string', 'number', 'boolean'].includes(typeof result) || result === null) {
       return;
     }
+    if (result === undefined) {
+      throw new Error(`result was undefined`);
```

Comment:
can we have a better Error here?

Reply:
Did you mean a better message?

Otherwise, it's tricky, we're at some point in a recursive function traversing a data structure, I think we'd want to rebuild it and mark the offending location, which doesn't really seem worth the effort yet.